src/pages/listing/BookingCard.tsx

```diff
@@ -110,6 +122,51 @@ const BookingCard = ({
       return utcDay.isBetween(startDate, endDate, undefined, pickingEnd ? '(]' : '[)');
     }).length >= quantity;
   }
+
+  async function startBooking() {
+    setBooking(true);
+    try {
+      const { data } = await createBooking({
+        checkInDate: String(startDate),
+        checkOutDate: String(endDate),
+        listingId: id,
+        numberOfGuests: numberOfGuests
+      });
+      history.push(`/bookings/${data.createBooking.id}`);
```

Comment:
i think for now we have to do `window.location.href` because history.push uses react router from what i remember

Reply:
Ah, think that explains why I needed to reload to see the booking change. Thanks!